### DISCUSSION TITLE
.config/zsh instead of .zsh

### DISCUSSION THREAD

**Discussion by 2Belette** [zimfw/zimfw#462] [General]
just wondering if there is any impact somehow ? it seems that this is coming from one of the latest version of ssh where it stores config in .config and not in .zsh anymore

thanks

**Comment by Plloi** [comment]
Do you have a reference for that config location change? i'm not aware of that being the case?

**Comment by cattyhouse** [comment]
@2Belette @Plloi  /etc/zsh/zshenv mentions : if $HOME does not have any of these files `.zshenv, .zprofile, .zshrc, .zlogin`, then it will use `~/.config/zsh` as config path.

solution: `touch ~/.zshenv`
